packages/components/src/popover/index.js

```diff
@@ -246,6 +246,7 @@ const Popover = ( {
 	__unstableSlotName = SLOT_NAME,
 	__unstableAllowVerticalSubpixelPosition,
 	__unstableAllowHorizontalSubpixelPosition,
+	__unstableFixedPosition = true,
```

Comment:
I'm a bit concerned about the growing number of unstable props for this component. This makes me think there's an audit work that's needed to rethink/consolidate these props. It becomes very hard to understand how this component works.

The "modifiers" approach `popper.js` uses could absorb a lot of these I think.

Reply:
Yes, it's concerning, but I don't see an easy way around it for now. These unstable props allow us to experiment with the popover without committing to an API.

* `__unstableSticky` can probably be stabilised at some point.
* `__unstableSlotName` should ideally go away.
* `__unstableAllowVerticalSubpixelPosition` and `__unstableAllowHorizontalSubpixelPosition` when Chrome fixes the positioning bug (they're working on it), or if we can find a different workaround.
* `__unstableFixedPosition` could be stabilised eventually.

Reply:
Minor: Noting that "could be stabilized" is meant to be a differentiator between what we consider to be "experimental" vs. "unstable".

>An unstable API is one which serves as a means to an end. It is not desired to ever be converted into a public API.

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis

Reply:
They all do serve as means to an end. If it's interesting for other popovers, we can consider making it experimental or stable though. I think we should avoid marking anything stable or experimental if we're thinking about using a third party library.